gherkin-utils/java/src/main/java/io/cucumber/gherkin/utils/DefaultHandlers.java

```diff
@@ -0,0 +1,20 @@
+package io.cucumber.gherkin.utils;
+
+import io.cucumber.messages.Messages;
+
+public abstract class DefaultHandlers {
```

Comment:
This can be an interface with default methods. It should avoid the need to create a subclass each time an instance is needed. 

It should not be named `Default` because it is an interface and not a concrete implementation. So `Handler` (singular) should do.

@aslakhellesoy this an implementation of the visitor pattern. Any reason its not named as such in JavaScript?

Reply:
hi @mpkorstanje,
Ok I will replace my abstract class ```DefaultHandlers``` to interface ```Handler```. I try to use naming conventions similar to JavaScript version, @aslakhellesoy pls give some comments about it.